Public/Src/Tools/Execution.Analyzer/BxlAnalyzer.dsc

```diff
@@ -13,13 +13,8 @@ namespace Execution.Analyzer {
         generateLogs: true,
         rootNamespace: "BuildXL.Execution.Analyzer",
         skipDocumentationGeneration: true,
-        sources: [
-            ...globR(d`.`, "*.cs"),
-            ...GrpcSdk.generate({
-                proto: globR(d`.`, "*.proto"),
-                includes: [importFrom("Google.Protobuf.Tools").Contents.all],
-            }).sources,
-        ],
+        sources: [...globR(d`.`, "*.cs")],

```

Comment:
No need to create an array and then to a single ... (addRange). 
you can simply write:

```sources: globR(d`.`, "*.cs"),```